### ISSUE TITLE
Exception with locust 2.2.1

### ISSUE THREAD

**Issue by dininski** [myzhan/locust4j#35]
When running with the latest version of locust (2.2.1) the following exception occurs:

```
15:24:07.087 TKD [Thread-3receive-from-client] ERROR c.g.myzhan.locust4j.runtime.Runner - Error while receiving a message
java.io.IOException: Message received unsupported type: ARRAY
	at com.github.myzhan.locust4j.message.Message.unpackMap(Message.java:88)
	at com.github.myzhan.locust4j.message.Message.unpackMap(Message.java:85)
	at com.github.myzhan.locust4j.message.Message.<init>(Message.java:38)
	at com.github.myzhan.locust4j.rpc.ZeromqClient.recv(ZeromqClient.java:42)
	at com.github.myzhan.locust4j.runtime.Runner$Receiver.run(Runner.java:387)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

From a little bit of debugging it appears that there is a `user_classes` message being passed of type `ARRAY`

**Comment by myzhan** [comment]
I see, since 2.2.0, locust sends parsed_options to workers. I'm working on a new bugfix version of locust4j.

see alse:
https://github.com/locustio/locust/issues/1877

**Comment by cyberw** [comment]
Does locust4j have to match the exact layout of the message? I try to be careful in removing/changing the payload in a way that would require changes in worker implementations, but I didnt know you'd have issues with _new_ parameters.

**Comment by stephen-harris** [comment]
@myzhan Just created a PR (had fixed it before I'd seen your message about working on a fix, so feel free to disregard).

@cyberw The issue was less additional data in the message, and more to do with the type of data (array) that hadn't previously been supported.

**Comment by myzhan** [comment]
@stephen-harris Thanks for your effort, I will review your MR.

**Comment by myzhan** [comment]
FYI, I have released locust4j 2.2.0 to address this issue.
